docs/localInstallation.md

```diff
@@ -583,201 +608,180 @@ ws.layers.dir=/var/www/html/layers
 
-_You can go directly with http://127.0.0.1:8080/phis2ws __
+**Note** </br>
+You can directly go on the web service with the URL http://127.0.0.1:8080/phisws
 
-To check you are correctly configurate your web service:
-Go to **Documentation link**, try **brapiv1token**:
-post -> clic on example -> try it out  
-If you haven't:
-```
-Response Code
+To check if you have correctly set up your web service :
+ - Click on the `Documentation link`
+ - Find the line `brapiv1token` and click on it
+ - Click on `POST`
+ - Click on the example field
+ - Click on the `Try it out!` button  
 
+The `Response Code` you get should be:
+```
 201
 ```
-please go to [current error](#errors-with-th-web-service) section.
-
 
+Otherwise, please go to the [current error](#errors-with-th-web-service) section.
 
 ### Web application
 
-
 #### Folder
 
-
-The web application deployment is did by apache2. You have to copy webapp folder in apache root folder in our case /var/www/html.
-```
-  sudo cp -r ~/Phis/phis-webapp /var/www/html
+The web application deployment is done by Apache2. You have to copy the webapp folder in the Apache root folder (in our case `/var/www/html`).
+```bash
+sudo cp -r <GitHub folder>/phis-webapp /var/www/html
 ```
-Change right on this folder:
+Change the permissions of this folder:
 ```
-  sudo chown -R [username]:www-data /var/www/html/phis-webapp
-  sudo chmod 775 -R /var/www/html/phis-webapp
+sudo chown -R <username>:www-data /var/www/html/phis-webapp
+sudo chmod 775 -R /var/www/html/phis-webapp
 ```
-*information:  
-www-data is the default apache2 username. But in rare case it can be a different name, you can check that in /etc/apache2/envars file.*
-
+**Note**</br>
+`www-data` is the default Apache2 username. But in rare case it can be different. You can check the value of `APACHE_RUN_USER` it in the `/etc/apache2/envars` file to be sure.
 
 #### Configuration
 
-
-To deploy in localhost it isn't necessary to open webapp with netbeans , you only need adapt configuration files so you can use classical editor.  
-Netbeans users:  
-Open the webapp folder with netbeans, like a php project.
-
-open project -> **/var/www/html/phis-webapp** -> open  
-Configuration files are in: *phis-webapp* -> *sources files* -> *config*  
-Other users:  
-Configuration files are in: **/var/www/html/phis-webapp/config**   
-
-Edit:
-  - *webservices.php*  
-Adapt the last line with the correct URL in our case is:
-**http://127.0.0.1:8080/phis2ws/rest**
-
-If you are all exactly doing like in this document, it's the only configuration file you need to change. But you can check other configuration files: compare paths and URL with information written in web services configuration files.
-
-
+To deploy the web application in localhost, it isn't necessary to open it with Netbeans:
+- Edit `/var/www/html/phis-webapp/config/web_services.php`
+- Set the value of `WS_PHIS_PATH_DOC`to `http://127.0.0.1:8080/phis2ws/rest`
```

Comment:
the phisws must be the same in the line 632 (http://127.0.0.1:8080/phisws) and this line (http://127.0.0.1:8080/phis2ws/rest). Which one is the right url ?

Reply:
I would say http://127.0.0.1:8080/phisws

Reply:
Than it might be this url everywhere ;)